src/scripts/update_funding_round.js

```diff
@@ -51,6 +52,7 @@ const main = async () => {
         curRoundVoteEnd = curRound.get('Voting Ends')
         curRoundVoteType = curRound.get('Vote Type')
         curRoundBallotType = curRound.get('Ballot Type')
+        await processAirtableProposalStandings(curRoundNumber) // process proposal standings
```

Comment:
Hi Berkay,

Thanks for updating this and getting all the functionality in the same place.

99.9% of the time, there should be a curRound available, so this block mainly serves to initialize the data. I think the function could sit outside of this initialization, but it's a minor detail, and don't think it requires any change.

Thanks for merging the functionality together!